packages/app/src/common/components/notifications.tsx

```diff
@@ -46,11 +46,7 @@ export const notifyTransaction = (n: {
               {n.txid !== undefined ? (
                 <div className="flex flex-row text-outset">
                   <a
-                    href={
-                      "https://explorer.solana.com/tx/" +
-                      n.txid +
-                      `?cluster=mainnet`
-                    }
+                    href={"https://solana.fm/tx/" + n.txid + `?cluster=mainnet`}
```

Comment:
Would be cool if it was the cluster the app currently runs on

Reply:
true- will fix in another PR. There are a few places where we assume mainnet so prob should fix together